support-frontend/assets/helpers/globalsAndSwitches/landingPageSettings.ts

```diff
@@ -17,6 +17,28 @@ export interface TickerSettings {
 	name: TickerName;
 }
 
+interface CountdownTheme {
+	backgroundColor: string;
+	foregroundColor: string;
+}
+interface CountdownSettings {
+	label: string;
+	countdownStartInMillis: string;
+	countdownDeadlineInMillis: string;
+	theme: CountdownTheme;
+}
+
+export type CountdownSetting = {
```

Comment:
The model name here is quite confusing.
Since this model is almost the same, and the timestamp fields in `CountdownSettings` need to be renamed, how about:
```
interface CountdownSettings {
	label: string;
	countdownStartTimestamp: string;
	countdownDeadlineTimestamp: string;
	theme: CountdownTheme;
}

type ParsedCountDownSettings = CountdownSettings & {
	countdownStartInMillis: number;
	countdownDeadlineInMillis: number;
}
```